interface/main/tabs/js/tabs_view_model.js

```diff
@@ -331,10 +331,12 @@ function menuActionClick(data,evt)
           dataurl = '/interface/patient_file/encounter/encounter_top.php?formname=' +
             matches[1] + '&formdesc=' + encodeURIComponent(data.label());
         }
-
+        
+        if(!openExistingTab(data)){
         navigateTab(webroot_url + dataurl, data.target, function () {
             activateTabByName(data.target,true);
         },xl("Loading") + " " + dataLabel);
+    }
 
         var par = $(evt.currentTarget).closest("ul.menuEntries");
         par.wrap("<ul class='timedReplace' style='display:none;'></ul>");
```

Comment:
Maybe `style='display:none;'`  can change to the Bootstrap class `d-none`.

Reply:
Sorry But that line is not covered in my changes 😕. Can I still alter @bradymiller ?

Reply:
In this case, would keep the PR focused (I am saying that since I have seen that `d-none` and `style='display:none;` do not always end up doing the same thing; ie. avoid adding issue to your fix)

Reply:
Okay, thanks for your explanation.